Pear/Controller/ChatViewController.swift

```diff
@@ -321,7 +321,7 @@ extension ChatViewController: UITableViewDataSource {
         let message = groupedMessagesByDate[indexPath.section][indexPath.row]
         cell.configure(for: message, user: currentUser, pair: messageUser)
         cell.viewProfile = {
-            self.navigationController?.pushViewController(ProfileViewController(user: self.currentUser, otherUserId: self.messageUser.id), animated: true)
+            self.navigationController?.pushViewController(ProfileViewController(user: self.currentUser, viewType: .otherUser, otherUserId: self.messageUser.id), animated: true)
```

Comment:
Is `otherUserId` a required field when `viewType` is not `.otherUser`? If so, you could wrap the value for `otherUserId` as an associated value of the `.otherUser` enum constructor.